src/ess_rhat.jl

```diff
@@ -195,25 +200,23 @@ function mean_autocov(k::Int, cache::BDAESSCache)
 end
 
 """
-    ess_rhat(
-        [estimator,]
+    ess(
         samples::AbstractArray{<:Union{Missing,Real},3};
+        type=:bulk,
+        [estimator,]
         method=ESSMethod(),
         split_chains::Int=2,
         maxlag::Int=250,
+        kwargs...
     )
 
-Estimate the effective sample size and ``\\widehat{R}`` of the `samples` of shape
+Estimate the effective sample size (ESS) of the `samples` of shape
 `(draws, chains, parameters)` with the `method`.
 
-By default, the computed ESS and ``\\widehat{R}`` values correspond to the estimator `mean`.
-Other estimators can be specified by passing a function `estimator` (see below).
+Optionally, only one of the `type` of ESS estimate to return or the `estimator` for which
+ESS is computed can be specified (see below). Some `type`s accept additional `kwargs`.
```

Comment:
I was a bit confused by the first sentence. Maybe it could be reformulated a bit, maybe something like
```suggestion
Optionally, one can specify either the `type` of the ESS estimate or the `estimator` for which
ESS is computed (see below). Some `type`s accept additional `kwargs`.
```